### PR TITLE
Add ActiveAdmin information to README.md

### PR DESCRIPTION
**Pull request by beasrodrigues24**
Why:
* It wasn't clear what requires customization

### CODE REVIEW

**Comment by beasrodrigues24** [comment]
@jreis13 @RicardoVaz1 Check if I forgot something please :pray: !

File: README.md [hunk 1]
```diff
@@ -40,3 +40,19 @@ Currently the frontend is hosted in https://subvisuwheel.netlify.app/. To run lo
 ### Eslint, Prettier and Rubocop 
 
 `./bin/lint`
+
+### Active Admin customization
+
+This app allows for some customization, hosted in https://subvisuwheel.herokuapp.com/admin. You can create an admin account by running the console with `heroku run bin/rails c -a subvisuwheel` and then `AdminUser.create(email: "email_of_choice, password: password_of_choice")`.
```
Comment (RicardoVaz1) [inline README.md:46]: Should be: `AdminUser.create(email: "email_of_choice", password: "password_of_choice")`

**Review by RicardoVaz1 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (RicardoVaz1) [reply README.md:46]: After changing this you can :shipit:

Comment (beasrodrigues24) [reply README.md:46]: Well noticed! Thanks

**Pull request merged**